packages/core/test/Asset/util/isValidHandle.test.ts

```diff
@@ -0,0 +1,26 @@
+import { Asset } from '../../../src';
+
+const validHandles = [
+  'bob',
+  'a',
+  'alice',
+  'test-handle',
+  'test.handle',
+  'test-handle-123',
+  'test-handle-123.456',
+  'handle_name'
+];
+
+const invalidHandles = ['bob!', '$wallet', 'alice@', 'test*handle#2', 'lace&bob', 'ada%1_test'];
+
+describe('isValidHandle', () => {
+  test('returns false for empty string', () => {
+    expect(Asset.util.isValidHandle('')).toBe(false);
+  });
+  test.each(validHandles)('returns true for valid handles', (handle) => {
```

Comment:
```suggestion
  test.each(validHandles)('returns true for valid handle "%s"', (handle) => {
```

This would make the output of `yarn test --verbose` a bit more clear as the failing tests description will immediately report the failing _handle name_.

Reply:
06e824b should resolve this